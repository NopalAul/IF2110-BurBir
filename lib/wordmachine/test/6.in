6
12412412412;
