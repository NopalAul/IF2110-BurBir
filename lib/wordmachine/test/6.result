lib/wordmachine/test/6.in lib/wordmachine/test/6.out: WRONG
