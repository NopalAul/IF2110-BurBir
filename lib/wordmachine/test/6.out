BALAS
BALAS
1
BALAS
1
2
CARI_KICAUAN
#TEST6
